requirements.txt

```diff
@@ -5,6 +5,10 @@ keyring==5.7.1; python_version < '2.7'
 keyring<=9.1; python_version >= '2.7'
 keyrings.alt; python_version >= '2.7'
 
+# GitDB 4.0.1 no longer supports Python 2.6
+gitdb==0.6.4; python_version < '2.7'
```

Comment:
`==2.0.4` should work fine according to https://pypi.org/project/gitdb2/2.0.4/?

How come this only starts failing now though?

Reply:
As mentioned, 4.0.1 was released on 23rd Feb. And you linked gitdb**2** but we(GitPython) need gitdb

Reply:
Ah, my bad...